pom.xml

```diff
@@ -163,6 +166,65 @@
         </annotationProcessorPaths>
       </configuration>
     </plugin>
+    <plugin>
+      <groupId>org.codehaus.mojo</groupId>
+      <artifactId>exec-maven-plugin</artifactId>
+      <version>${org.codehaus.mojo.exec.plugin.version}</version>
+      <configuration>
+        <mainClass>com.redhat.rhjmc.containerjfr.ContainerJfr</mainClass>
+        <additionalClasspathElements>
+          <additionalClasspathElement>${project.build.directory}/assets/app/resources</additionalClasspathElement>
+        </additionalClasspathElements>
+      </configuration>
+      <executions>
+        <execution>
+          <id>start-container</id>
+          <phase>pre-integration-test</phase>
+          <goals>
+            <goal>exec</goal>
+          </goals>
+          <configuration>
+            <executable>${imageBuilder}</executable>
+            <arguments>
+              <argument>run</argument>
+              <argument>--hostname=container-jfr</argument>
+              <argument>--name=container-jfr-itest</argument>
+              <argument>--publish</argument>
+              <argument>9091:9091</argument>
+              <argument>--publish</argument>
+              <argument>${containerjfr.itest.webPort}:${containerjfr.itest.webPort}</argument>
+              <argument>--env</argument>
+              <argument>CONTAINER_JFR_LOG_LEVEL=ALL</argument>
+              <argument>--env</argument>
+              <argument>CONTAINER_JFR_WEB_HOST=0.0.0.0</argument>
+              <argument>--env</argument>
+              <argument>CONTAINER_JFR_LISTEN_HOST=0.0.0.0</argument>
+              <argument>--env</argument>
+              <argument>CONTAINER_JFR_WEB_PORT=${containerjfr.itest.webPort}</argument>
+              <argument>--env</argument>
+              <argument>CONTAINER_JFR_EXT_WEB_PORT=${containerjfr.itest.webPort}</argument>
+              <argument>--detach</argument>
+              <argument>--rm</argument>
+              <argument>${containerjfr.imageStream}:latest</argument>
```

Comment:
Should we use `${containerjfr.imageVersion}` instead of `latest` here? Also I had this task hang for me, but couldn't reproduce it. Not sure why.

```
INFO] --- jib-maven-plugin:2.1.0:dockerBuild (build-oci-image) @ container-jfr ---
[WARNING] Setting image creation time to current time; your image may not be reproducible.
[INFO] 
[INFO] Containerizing application to Docker daemon as quay.io/rh-jmc-team/container-jfr, quay.io/rh-jmc-team/container-jfr:0.16.0...
[WARNING] Base image 'gcr.io/distroless/java:11' does not use a specific image digest - build may not be reproducible
[INFO] Using base image with digest: sha256:c94feda039172152495b5cd60a350a03162fce4f8986b560ea555de4d276ce19
[INFO] 
[INFO] Container entrypoint set to [java, -XX:+CrashOnOutOfMemoryError, -Dcom.sun.management.jmxremote.port=9091, -Dcom.sun.management.jmxremote.ssl=false, -Dcom.sun.management.jmxremote.authenticate=false, -cp, /app/resources:/app/classes:/app/libs/*, com.redhat.rhjmc.containerjfr.ContainerJfr]
[INFO] 
[INFO] Built image to Docker daemon as quay.io/rh-jmc-team/container-jfr, quay.io/rh-jmc-team/container-jfr:0.16.0
[INFO] Executing tasks:
[INFO] [==============================] 100.0% complete
[INFO] 
[INFO] 
[INFO] --- exec-maven-plugin:1.6.0:exec (start-container) @ container-jfr ---
^C
```

Reply:
Sure, I'll push a commit momentarily to use `imageVersion` rather than `latest`.

I've seen the same hang on start-container. I just filed an issue for it (#134). I think it's a podman thing, but I'm going to test that hypothesis by running the suite a few times using docker instead and see what happens.